### PR TITLE
Improvements to schema stitching

### PR DESCRIPTION
**Pull request by orta**
- Adds support for introspecting metaphysics' current runtime via a query, similar to how we can look up vars for convection via the config query:

screen shot 2018-09-25 at 6 04 35 pm

- Switches the imports around in order to accurately handle a stitched schema when dumping, and loosens some of the ENV requirements

  - It now just looks for a truth values for stitching ENV vars 
  - Moved some of the imports around so that the default `src/schema` can now return the stitched MP schema so tooling will work

### CODE REVIEW

File: src/lib/stitching/mergeSchemas.ts [hunk 1]
```diff
@@ -5,35 +5,37 @@ import { consignmentStitchingEnvironment } from "lib/stitching/convection/stitch
 import { executableExchangeSchema } from "lib/stitching/exchange/schema"
 import config from "config"
 
-import localSchema from "../../schema"
+import localSchema from "../../schema/schema"
```
[the remark this answers is not in the document]
Comment (orta) [reply src/lib/stitching/mergeSchemas.ts]: Yeah, it is, but everyone else only needs to import the index, calling it "localSchema" would probably confuse folks

[the remark this answers is not in the document]
Comment (damassi) [reply src/lib/stitching/mergeSchemas.ts]: Maybe `import localSchema from 'schema/schema'`?

File: src/schema/services/metaphysics.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+// so that you can validate the env at runtime easily.
+
+const mapEnvBooleans = {
+  environment: !!config.NODE_ENV,
+  queryTracing: !!config.ENABLE_QUERY_TRACING,
+  heapDumps: !!config.ENABLE_HEAPDUMPS,
+  stitching: !!config.ENABLE_SCHEMA_STITCHING,
+  stitchingGravity: !!config.ENABLE_GRAVQL_ONLY_STITCHING,
+  stitchingConvection: !!config.ENABLE_CONSIGNMENTS_STITCHING,
+  stitchingExchange: !!config.ENABLE_ECOMMERCE_STITCHING,
```
Comment (damassi) [inline src/schema/services/metaphysics.ts]: 👍

**Comment by orta** [comment]
Cool, yep - merge on green

**Pull request merged**